src/main/java/com/hotels/bdp/circustrain/bigquery/conversion/BigQueryToHivePartitionConverter.java

```diff
@@ -69,7 +69,8 @@ public Partition convert() {
 
   // Work around for issue: https://issues.apache.org/jira/browse/HIVE-18767
   // Delete this code once the fix
-  // (https://github.com/apache/hive/commit/2fe5186a337141b6fd80b40abbc8bc4226bee962#diff-2a1b7c6ec7a77f1ca9ad84225d192e36)
+  // (https://github.com/apache/hive/commit/2fe5186a337141b6fd80b40abbc8bc4226bee962#
```

Comment:
Is there no way we can squeeze this onto one line so you can directly copy/paste and then view it  in a browser? Perhaps remove the empty space after each // and then get rid of the brackets around the link? will that get it just into 120 chars?

Reply:
Hmm I think the space needs to be after `//`. But I could remove the `https://` because the link pasted like that is fine in a web browser.